client/gutenberg/editor/api-middleware/index.js

```diff
@@ -17,6 +17,15 @@ import wpcom from 'lib/wp';
 const debug = debugFactory( 'calypso:gutenberg' );
```

Comment:
All the changes to this file are contained in #30332. Please provide review there.